### PR TITLE
Upgrade to newest Alerts srcdep

### PR DESCRIPTION
**Pull request by ppalaga**

### CODE REVIEW

**Comment by ppalaga** [comment]
@jpkrohling could you please review and eventually merge?

File: pom.xml [hunk 1]
```diff
@@ -82,13 +82,13 @@
...
+    <version.org.hawkular.accounts>2.0.10.Final</version.org.hawkular.accounts>
+    <version.org.hawkular.agent>0.15.2.Final</version.org.hawkular.agent>
+    <version.org.hawkular.alerts>0.8.0.Final-SRC-revision-2b05b4acbae7c7f45b3e6f19a4ca3c4bc60ca9e9</version.org.hawkular.alerts>
+    <version.org.hawkular.cmdgw>0.10.7.Final</version.org.hawkular.cmdgw>
     <version.org.hawkular.commons>0.3.2.Final</version.org.hawkular.commons>
-    <version.org.hawkular.cmdgw>0.10.6.Final</version.org.hawkular.cmdgw>
-    <version.org.hawkular.metrics>0.11.0.Final-SRC-revision-7a90403c761cc446b1bc357eaac351d58ded00f4</version.org.hawkular.metrics>
-    <version.org.hawkular.inventory>0.11.0.Final</version.org.hawkular.inventory>
+    <version.org.hawkular.inventory>0.12.0.Final</version.org.hawkular.inventory>
+    <version.org.hawkular.metrics>0.10.0.Final</version.org.hawkular.metrics>
```
Comment (jpkrohling) [inline pom.xml]: The previous src-deps was 0.11.... , and you are using 0.10.... 

Is this intended, or did you mean 0.11 ?

**Comment by jpkrohling** [comment]
Other than the comment above, it looks sane to me.

File: pom.xml (hunk 1 above)
Comment (ppalaga) [inline pom.xml]: Good catch, thanks!

**Comment by ppalaga** [comment]
b1b8310 is fixing the metrics version.

@jpkrohling please merge.

**Pull request merged**